src/components/ScenarioParameters/components/ScenarioParametersTabs.js

```diff
@@ -92,6 +92,7 @@ const ScenarioParametersTabs = ({
             visibleTabs.map((tab, index) => (
               <Tab
                 key={index}
+                data-cy={tab.value}
```

Comment:
Could we add a prefix or suffix to have more info in the data-cy than just the parameters group name ? (e.g. `tab.value + '_tab'`)